tests/object/test_object.py

```diff
@@ -41,26 +41,25 @@ def test_show(
 
 
 DESCRIBE_TEST_OBJECTS = [
-    ("compute-pool", "compute-pool-example"),
-    ("network-rule", "network-rule-example"),
-    ("integration", "integration"),
-    ("network-rule", "network rule"),
-    ("database", "database-example"),
-    ("function", "function-example"),
-    # ("job", "job-example"),
-    ("procedure", "procedure-example"),
-    ("role", "role-example"),
-    ("schema", "schema-example"),
-    ("service", "service-example"),
-    ("secret", "secret-example"),
-    ("stage", "stage-example"),
-    ("stream", "stream-example"),
-    ("streamlit", "streamlit-example"),
-    ("table", "table-example"),
-    ("task", "task-example"),
-    ("user", "user-example"),
-    ("warehouse", "warehouse-example"),
-    ("view", "view-example"),
+    ("compute-pool", "compute_pool_example"),
```

Comment:
Technically hyphens (`-`) are not allowed as unquoted identifiers, so we should not be using them as arguments for our test object names. This currently cannot be validated with `is_valid_object_name` because there are also cases where the object identifier is given as object_name(argDataType1, argDataType2, ...) such as UDFs and procedures. Future work could be to add this validation.